config/defaults.mk

```diff
@@ -106,6 +106,7 @@ MFEM_USE_PETSC       = NO
 MFEM_USE_MPFR        = NO
 MFEM_USE_SIDRE       = NO
 MFEM_USE_CONDUIT     = NO
+MFEM_USE_SCOREC      = NO
```

Comment:
Document `MFEM_USE_SCOREC` in the `INSTALL` file.

Reply:
Documention for `MFEM_USE_PUMI` is added to the `INSTALL` file. 
https://github.com/mfem/mfem/blob/pumi-dev/INSTALL#L352.
We replace all `MFEM_USE_SCOREC` with `MFEM_USE_PUMI`. Following the previous comment.